disco/types/guild.py

```diff
@@ -311,38 +354,49 @@ class Guild(SlottedModel, Permissible):
         All of the guild's voice states.
     premium_tier : int
         Guild's premium tier.
-    premium_subscription_count: int
+    premium_subscription_count : int
         The amount of users using their nitro boost on this guild.
     """
     id = Field(snowflake)
-    owner_id = Field(snowflake)
-    afk_channel_id = Field(snowflake)
-    embed_channel_id = Field(snowflake)
-    system_channel_id = Field(snowflake)
     name = Field(text)
     icon = Field(text)
     splash = Field(text)
-    banner = Field(text)
+    owner = Field(bool)
+    owner_id = Field(snowflake)
+    permissions = Field(int)
     region = Field(text)
+    afk_channel_id = Field(snowflake)
     afk_timeout = Field(int)
     embed_enabled = Field(bool)
+    embed_channel_id = Field(snowflake)
     verification_level = Field(enum(VerificationLevel))
-    explicit_content_filter = Field(enum(ExplicitContentFilterLevel))
     default_message_notifications = Field(enum(DefaultMessageNotificationsLevel))
-    mfa_level = Field(int)
-    features = ListField(str)
-    members = AutoDictField(GuildMember, 'id')
-    channels = AutoDictField(Channel, 'id')
+    explicit_content_filter = Field(enum(ExplicitContentFilterLevel))
     roles = AutoDictField(Role, 'id')
     emojis = AutoDictField(GuildEmoji, 'id')
-    voice_states = AutoDictField(VoiceState, 'session_id')
+    features = ListField(str)
+    mfa_level = Field(int)
+    application_id = Field(snowflake)
+    widget_enabled = Field(bool)
+    widget_channel_id = Field(snowflake)
+    system_channel_id = Field(snowflake)
+    joined_at = Field(datetime)
+    large = Field(bool)
+    unavailable = Field(bool)
     member_count = Field(int)
-    premium_tier = Field(int)
-    premium_subscription_count = Field(int, default=0)
-    vanity_url_code = Field(text)
-    max_presences = Field(int)
+    voice_states = AutoDictField(VoiceState, 'session_id')
+    members = AutoDictField(GuildMember, 'id')
+    channels = AutoDictField(Channel, 'id')
+    presences = Field(enum(Presence))
```

Comment:
Why is this an enum field? `presences` are a dict of Member Presences. These are used to update the user in the state.

Reply:
Already patched, not meant to be there at all. Breaks presences altogether. Was tired while updating that bit.